plugins/kotlin/uast/uast-kotlin-base/test/org/jetbrains/uast/test/common/kotlin/UastResolveEverythingTestBase.kt

```diff
@@ -32,4 +34,31 @@ interface UastResolveEverythingTestBase : UastPluginSelection, UastFileCompariso
             kind = "resolved"
         )
     }
+
+    fun renderLightElementDifferently(element: PsiElement?): String {
+        var str = element.toString()
+        if (!isFirUastPlugin || element !is KtLightElement<*, *>) return str
+        // NB: all declarations in FIR LC are internal, so type checking won't work.
+        TAG.forEach { (firLC, uLC) -> str = str.replace(firLC, uLC) }
+        REGEX.forEach { (firRegex, uLC) -> str = firRegex.replace(str, uLC) }
+        return str
+    }
+
+    companion object {
+        private val TAG: Map<String, String> = mapOf(
+            "FirLightAnnotationClassSymbol" to "KtUltraLightClass",
```

Comment:
Let's do not relay on implementation details :) We may use some more general names here, like KtLightClass or or KtLightMethod. Or maybe use some fancy string representation like `Kotlin Light Class`

Reply:
I see. In that case, you meant mapping each LC tag to one central, impl-agnostic name? E.g., "FirLightAnnotationClassSymbol" -> "Kotlin Light Class", "KtUltraLightClass" -> "Kotlin Light Class", and so on?

Reply:
Yes

Reply:
Done at 2c1129d. Somehow, I decided to avoid space (` `) in tags.